packages/harpguru-core/src/hooks/use-sizes/use-sizes.ts

```diff
@@ -1,7 +1,8 @@
 import { useLayoutFacts } from '../use-layout-facts'
-import { SizeScheme, SizeSchemes } from '../styles-types'
 import { getWindowDimensions } from '../../packages/get-window-dimensions'
 
+import { SizeScheme, SizeSchemes } from './use-sizes-types'
+
 // Here's what we need this hook to provide:
```

Comment:
I think perhaps I failed to identify this as a comment & plan which needs removing in the techdebt. Proabably should have been spotted in #107 .

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/113#issuecomment-922351622

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.